scripts/scripts.js

```diff
@@ -179,6 +179,17 @@ export function buildImageWithCaptionBlocks(main, buildBlockFunction) {
   });
 }
 
+function buildDownloadableLinks(main) {
```

Comment:
I think this should be a part of the anchor decoration flow i.e. `decorateAnchors/decorateExternalAnchors`. As a part of anchor decoration, you can invoke this method to decorate the downloadable links specially.

Reply:
Done. Can you please check now